schemas/WebpackOptions.json

```
@@ -1159,11 +1159,12 @@
           "description": "The maximum EcmaScript version of the webpack generated code (doesn't include input source code from modules).",
           "anyOf": [
             {
-              "enum": [5]
+              "enum": [5, 6, 7, 8, 9, 10, 11]
             },
             {
               "type": "number",
-              "minimum": 2015
+              "minimum": 2009,
```

2010, 2011, 2012, 2013, 2014 should not be allowed -> they don't exist and give incorrect results in normalizeEcmaVersion

```suggestion
              "minimum": 2015,
```

plus

``` js
{ "enum": [2009] }
```